packages/api/README.md

```diff
@@ -125,3 +125,16 @@ A few notes about the response.
 - The "SPAnonymous" app has `null` values for name and ref because it was not deployed with `/deploy`, but it's included so that `/list` provides a complete report of what paths are being made available.
 
 [mongo-mock]: https://github.com/williamkapke/mongo-mock
+
+### /apikeys
+
+| HTTP Method | Endpoint                     | Description                                                                  |
+| ----------- | ---------------------------- | ---------------------------------------------------------------------------- |
+| `POST`      | `/apikey`                    | Creates and returns an API key object such as `{ key: "ABCD" }` <sup>1</sup> |
+| `DELETE`    | `/apikey?hashedKey=e12e115a` | Deletes key ABCD (hashed to "e12e115a") <sup>2</sup>                         |
```

Comment:
It occurs to me that my initial recommendation for this REST interface may have been inconsistent, because `POST` and `DELETE` are being passed data in different ways.

`POST` expects a JSON payload in the body, and `DEELTE` expects a querystring param.  I'm thinking now that they should use the same approach.  Or, they should support both.

@sayak-sarkar I think the current approach is just fine for now, I'm just leaving this comment here to share what I'm thinking about future changes.

Reply:
@mwcz I agree. For now only the `POST` request expects a JSON payload while all the other endpoints essentially work with query strings. We should ideally decide on only a single approach eventually.